### PR TITLE
Updated: CLA.md

### PR DESCRIPTION
**Pull request by Atharva-Shinde**
**Which issue(s) this PR fixes**: #6401

Updated the CLA documentation for the new EasyCLA process.

### CODE REVIEW

File: CLA.md [hunk 1]
```diff
@@ -13,64 +13,53 @@ It is important to read and understand this legal agreement.
...
->
-> Hello,
->
-> You have signed CNCF Individual Contributor License Agreement.
-> You can see your document anytime by clicking View on HelloSign.
->
+#### 5. Validate your CLA
 
-Once you have this, the CLA authorizer bot will authorize your PRs.
+Once you are redirected to your GitHub Pull Request, enter `/easycla` to check whether your CLA was signed successfully or not.
```
Comment (mrbobbytables) [inline CLA.md:61]:
```suggestion
Once you are redirected back to your GitHub Pull Request, reply with a comment `/easycla` to update the CLA status of your PR.
```

File: CLA.md [hunk 2]
```diff
@@ -13,64 +13,53 @@ It is important to read and understand this legal agreement.
...
 
-#### 3. Complete signing process
+Once you select your type of contributor proceed to Sign the CLA and follow the instructions to complete the signing process through DocuSign.
 
-After creating your account, follow the instructions to complete the
-signing process through HelloSign.
+**Ensure your GitHub e-mail address matches e-mail address used to sign CLA**
 
-If you did not receive an email from HelloSign, [then request it here](https://identity.linuxfoundation.org/projects/cncf).
+After you have carefully filled up the information, Click "Finish" and you will be redirected to your GitHub Pull Request page.
```
Comment (mrbobbytables) [inline CLA.md:42]:
```suggestion
After you have filled out  the information, Click "Finish" and you will be redirected back to your Pull Request.
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks! I left a few suggestions that clean up some of the wording a little bit, but not hard blockers :)

**Comment by Atharva-Shinde** [comment]
Thanks @mrbobbytables for helping me out !! I have committed all the suggested changes :)

File: CLA.md [hunk 3]
```diff
@@ -13,64 +13,53 @@ It is important to read and understand this legal agreement.
...
+<img width="1407" alt="EasyCLA" src="https://user-images.githubusercontent.com/69111235/152224818-1246453a-b086-4a57-9d14-c10d62ad438f.png">
 
-If you are signing up as an employee, you must use your official
-person@organization.domain email address as your default GitHub
-address or configure it as an alias of your LF identify.
 
+After authorizing EasyCLA, you will be redirected to a page to identify which type of contributor you are. 
+Select the most appropriate option:
+  * Individual Contributor: To sign up as an individual or as an employee of a signed organization.
+  * Corporate Contributor: To sign as a corporation representative and manage signups from your organization.
```
Comment (mrbobbytables) [inline CLA.md:34]:
```suggestion
  * Individual Contributor: You are contributing as yourself, and not as part of another organization.
  * Corporate Contributor: You are contributing on behalf of your employer or other organization.
```

One last thing^^:;;
Missed it yesterday when reviewing.

Comment (Atharva-Shinde) [reply CLA.md]: done :)

**Comment by mrbobbytables** [comment]
Thanks! :) 
/approve
/unassign jdumars liggitt
/assign @alisondy 

For a 2nd set of eyes :)

/label tide/merge-method-squash
/hold

Holding till the change goes in on Saturday 🎉

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
/hold cancel

**Pull request merged**